docs/application-development/foundations.md

```diff
@@ -1622,17 +1622,6 @@ async def say_hello(name: str) -> str:
 
 In TypeScript, the return value is always a Promise.
 
-<<<<<<< HEAD
-
-````typescript
-import type * as activities from './activities';
-const { greet } = proxyActivities<typeof activities>({
-  startToCloseTimeout: '1 minute',
-});
-=======
-In the following example, `Promise<string>` is the return value.
```

Comment:
Do you want to retain this line? Or is the preceding line sufficient?

Reply:
Want the promise<string> line in the source file